config/settings.d/externalipam_netbox.yml.example

```diff
@@ -0,0 +1,3 @@
+---
+:url: 'http://my-netbox-url.com'
+:token: 'netbox_token'
```

Comment:
Missing newline, also is the URL really in the format of hostname without any path? Configuration files often do not explain this properly, a full path would be good to have if it is actually required. The same for phpIPAM.